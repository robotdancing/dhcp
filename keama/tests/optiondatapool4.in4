# option-data in DHCPv4 pool config

# authoritative is mandatory
authoritative;

# subnet declaration
subnet 10.5.5.0 netmask 255.255.255.224 {
    # pool declaration
    pool {
        # avoid empty pool
        range 10.5.5.5 10.5.5.10;
        # for a silly reason option-data is not allowed in DHCPv4 pools
	option domain-search "example.com", "example.org";
    }
}
