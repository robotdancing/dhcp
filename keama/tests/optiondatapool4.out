{
  # option-data in DHCPv4 pool config
  # authoritative is mandatory
  /// This configuration declares some subnets but has no interfaces-config
  /// Reference Kea #5256
  "Dhcp4": {
    "subnet4": [
      # subnet declaration
      {
        "id": 1,
        "subnet": "10.5.5.0/27",
        "pools": [
          # pool declaration
          {
            # avoid empty pool
            "pool": "10.5.5.5 - 10.5.5.10",
            "option-data": [
              # fixed now
              {
                "space": "dhcp4",
                "name": "domain-search",
                "code": 119,
//              "original-data": "\"example.com\", \"example.org\"",
                "data": "example.com, example.org"
              }
            ]
          }
        ]
      }
    ]
  }
}
